### PR TITLE
Pro 4073 image alignment

### PR DESCRIPTION
**Pull request by haroun**
## Summary

Allow gap cursor before and after inline images and tables

## What are the specific steps to test this change?

*For example:*

1. Add a new rich-text widget
2. Add a table or an image (you can add a space, select the space to make the toolbar visible)
3. Select the image or table and using the arrow up or down key, you should see a blinking cursor before or after the image/table
4. Add some text

Note: adding a inline image in an empty rich-text without setting a caption deletes the image, there is a ticket to track this

## What kind of change does this PR introduce?
*(Check at least one)*

- [x] Bug fix
- [ ] New feature
- [ ] Refactor
- [ ] Documentation
- [ ] Build-related changes
- [ ] Other

## Make sure the PR fulfills these requirements:

- [x] It includes a) the existing issue ID being resolved, b) a convincing reason for adding this feature, or c) a clear description of the bug it resolves
- [x] The changelog is updated
- [ ] Related documentation has been updated
- [ ] Related tests have been updated

If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: modules/@apostrophecms/rich-text-widget/ui/apos/tiptap-extensions/Image.js [hunk 1]
```diff
@@ -1,106 +1,128 @@
...
+} from '@tiptap/core';
 
 export default options => {
   return Node.create({
-
     name: 'image',
 
     addOptions() {
       return {
+        // inline: true,
```
[the remark this answers is not in the document]
Comment (haroun) [reply modules/@apostrophecms/rich-text-widget/ui/apos/tiptap-extensions/Image.js]: Removed

File: modules/@apostrophecms/rich-text-widget/ui/apos/components/AposRichTextWidgetEditor.vue [hunk 2]
```diff
@@ -213,6 +214,7 @@ export default {
     isVisuallyEmpty () {
       const div = document.createElement('div');
       div.innerHTML = this.value.content;
+      console.log(this.value, div.innerHTML, !div.textContent);
```
[the remark this answers is not in the document]
Comment (haroun) [reply modules/@apostrophecms/rich-text-widget/ui/apos/components/AposRichTextWidgetEditor.vue]: Removed

**Review by ValJed — APPROVED** [review]

File: modules/@apostrophecms/rich-text-widget/index.js [hunk 3]
```diff
@@ -435,7 +435,11 @@ module.exports = {
             },
             {
               tag: 'img',
-              attributes: [ 'src' ]
+              attributes: [ 'src', 'alt' ]
+            },
+            {
+              tag: 'figcaption',
```
Comment (boutell) [inline modules/@apostrophecms/rich-text-widget/index.js:441]: Is this a bug fix that warrants mention in the changelog?

File: CHANGELOG.md [hunk 4]
```diff
@@ -12,6 +12,10 @@
 
 ### Changes
 
+* Remove `stripPlaceholderBrs` and `restorePlaceholderBrs` from `AposRichTextWidgetEditor.vue` component.
```
Comment (boutell) [inline CHANGELOG.md:15]: Should mention the addition of alt attribute support for inline images.

**Review by boutell — APPROVED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (haroun) [reply CHANGELOG.md:16]: Yes

File: modules/@apostrophecms/rich-text-widget/index.js (hunk 3 above)
Comment (haroun) [reply modules/@apostrophecms/rich-text-widget/index.js]: Bit unclear if it was needed, I've removed this part to avoid side effect

**Pull request merged**